Autopilot/PathManager/Inc/CommWithAttitudeManager.hpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Communicates with the attitude manager to send commands and receive useful data.
+ * Author: Anthony Bertnyk
+ */
+
+#ifndef COMM_WITH_ATTITUDE_MANAGER_HPP
+#define COMM_WITH_ATTITUDE_MANAGER_HPP
+
+extern "C"
+{
+#include "cmsis_os.h"
```

Comment:
Put this in the .cpp file instead of the header file. Reason is that unit testing and simulation don't know jack shit about FreeRtos, yet someone may need to mock or simulate against your file someday. (That someone is me and that day will be Wednesday or Thursday LMAO).

Also, as a general rule, you'l want to have as few dependencies in your header files as possible. Makes things really modular and easy to understand.